### PR TITLE
[TASK] Rename Access module to Permissions

### PR DESCRIPTION
**Pull request by linawolf**

### CODE REVIEW

File: Documentation/PermissionsManagement/SettingUpBackendGroups/Index.rst [hunk 1]
```diff
@@ -123,7 +123,7 @@ Page Group
     selected branches of pages in the tree.
 
     Those groups will be assigned directly to the pages (see Page Permissions
-    for more details) following the TSConfig or the Access Module configuration.
+    for more details) following the TSConfig or the :guilabel:`Access` module configuration.
```
Comment (froemken) [inline Documentation/PermissionsManagement/SettingUpBackendGroups/Index.rst:126]: Can we change the name here, too?

**Review by froemken — COMMENTED** [review]
Maybe just one renaming missing

File: Documentation/PermissionsManagement/SettingUpBackendGroups/Index.rst (hunk 1 above)
Comment (linawolf) [inline Documentation/PermissionsManagement/SettingUpBackendGroups/Index.rst:126]:
```suggestion
    for more details) following the TSConfig or the :guilabel:`Permissions` module configuration.
```

Comment (linawolf) [reply Documentation/PermissionsManagement/SettingUpBackendGroups/Index.rst]: woops

**Review by froemken — APPROVED** [review]
Nice

**Pull request merged**